nat/traversal/pinger_test.go

```diff
@@ -110,6 +102,56 @@ func TestPinger_Provider_Consumer_Ping_Flow(t *testing.T) {
 	}, time.Second, 10*time.Millisecond)
 }
 
+func TestPinger_PingPeer_N_Connections(t *testing.T) {
+	pingConfig := &PingConfig{
+		Interval: 10 * time.Millisecond,
+		Timeout:  1000 * time.Millisecond,
+	}
+
+	provider := newPinger(pingConfig)
+	consumer := newPinger(pingConfig)
+
+	var pPorts, cPorts []int
+	ports, err := port.NewPool().AcquireMultiple(10)
+	assert.NoError(t, err)
+
+	for i := 0; i < 5; i++ {
+		pPorts = append(pPorts, ports[i].Num())
+		cPorts = append(cPorts, ports[5+i].Num())
+	}
+
+	go consumer.PingPeer("192.168.0.101", cPorts, pPorts, 2, 3)
```

Comment:
This ip `192.168.0.101` will probably not work for everyone.

Reply:
Wow, thanks. It's a leftover from debugging.